deploy/docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+  website:
+    build:
+      context: ../website
+      dockerfile: Dockerfile.website
+    container_name: enei-website
+    ports:
+      - "3333:3333"
+
+  website-nginx:
+    image: nginx:1-alpine
+    ports:
+      - "80:80"
+    volumes:
+      - ../website/nginx.conf:/etc/nginx/nginx.conf
```

Comment:
I probably wasn't really clear. The docker-compose stays at the root of the repository.
All files needed for docker deployment stay in the `deploy` folder, including the dockerfile. The dockerignore must be at the depository root. The context for the build of all dockerfiles is at the root of the repository. For now, this doesn't make a lot of sense, but in the future, if we have multiple apps or packages in this repository, it's easier to build them and their local dependencies.

Reply:
I believe that is now fixed